utils/build/rollup.config.js

```diff
@@ -62,79 +62,80 @@ function header() {
 
 const builds = [
 	{
-		input: 'src/Three.js',
+		input: {
+			'three.core.js': 'src/Three.core.js',
+			'three.module.js': 'src/Three.js',
+			'three.webgpu.js': 'src/Three.WebGPU.js',
+		},
 		plugins: [
 			glsl(),
 			header()
 		],
+		preserveEntrySignatures: 'allow-extension',
 		output: [
 			{
 				format: 'esm',
-				file: 'build/three.module.js'
+				dir: 'build',
+				minifyInternalExports: false,
+				entryFileNames: '[name]',
 			}
 		]
 	},
 	{
-		input: 'src/Three.WebGPU.js',
-		plugins: [
-			header()
-		],
-		output: [
-			{
-				format: 'esm',
-				file: 'build/three.webgpu.js'
-			}
-		]
-	},
-	{
-		input: 'src/Three.WebGPU.Nodes.js',
+		input: {
+			'three.core.js': 'src/Three.core.js',
+			'three.webgpu.nodes.js': 'src/Three.WebGPU.Nodes.js',
+		},
```

Comment:
This is split from building `three.webgpu.js` since Rollup emits hashed chunks, which are common with nodes. We transform core twice in isolating the two, but ideally we emit `three.core.js` Rollup would know this is external, and would not attempt to codesplit further.

Reply:
Actually, this won't work with Terser unless we hash or preserve mangled properties between builds.

Reply:
We were missing `minifyInternalExports` on the minified bundles. Now works like before.

Reply:
Leaving this here until I find a way to cleanup the emitted chunks. I think some tricky with `external` is needed so we don't try to transform `three.core.js` in the other files, but have them self-contained otherwise.